build/release/Makefile

```diff
@@ -55,29 +47,27 @@ ifeq ($(COMMIT_HASH),)
 override COMMIT_HASH := $(shell git rev-parse HEAD)
 endif
 
-REMOTE_NAME ?= origin
-
 PLATFORMS ?= $(ALL_PLATFORMS)
 
 # "helm" flavor is for pushing to s3/https endpoint
 # "charts" flavor is for pushing to oci endpoint
-ifneq ($(filter master release-%,$(BRANCH_NAME)),)
 FLAVORS ?= output images docs helm charts
-else
-FLAVORS ?= output
-override BRANCH_NAME := pr/$(BRANCH_NAME)
-endif
 
 DOCKER_REGISTRY ?= docker.io/rook
 QUAY_REGISTRY ?= quay.io/rook
 GHCR_REGISTRY ?= ghcr.io/rook
-REGISTRIES ?= $(DOCKER_REGISTRY) $(QUAY_REGISTRY) $(GHCR_REGISTRY)
+ifeq ($(TAGGED_RELEASE),false)
+# Publish always to dockerhub, including master and interim builds
+REGISTRIES = $(DOCKER_REGISTRY)
+else
+# Publish the images to all registries for tagged releases
+REGISTRIES = $(DOCKER_REGISTRY) $(QUAY_REGISTRY) $(GHCR_REGISTRY)
+endif
```

Comment:
Nit/opinion: it might be easier to read/maintain this code if this were moved down to be just above `define repo.targets` where it's used.

Reply:
It's also used in the helm chart section on line 156, so it's not as clear where to move it. I think it's nice to see variables defined together as much as possible earlier in the file so I'll leave it here for now.